src/Microsoft.AspNetCore.Server.IISIntegration/NativeMethods.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
-    internal class NativeMethods
+    internal unsafe class NativeMethods
```

Comment:
what's unsafe now? you only removed code.

Reply:
Ooops.